compile.go

```diff
@@ -263,17 +267,17 @@ func RegisterPackage(pkg *ast.Package) {
 }
 
 // RegisterPackageValue adds a value for an identifier in a builtin package
-func RegisterPackageValue(path, name string, value values.Value) {
+func RegisterPackageValue(pkgpath, name string, value values.Value) {
 	if finalized {
 		panic(errors.New("already finalized, cannot register builtin package value"))
 	}
-	packg, ok := stdlib.pkgs[path]
+	packg, ok := stdlib.pkgs[pkgpath]
 	if !ok {
-		packg = interpreter.NewPackage(name)
-		stdlib.pkgs[path] = packg
+		packg = interpreter.NewPackage(path.Base(pkgpath))
```

Comment:
This could possibly cause issues if we introduce sub-packages, but this works for right now and is likely changing soon once we have the `builtin` syntax.

Reply:
A package's name is not path, but rather the value of the `package <ident>` statement. The packages are keyed in the map by their path which is unique. The package is name is used as the default for the identifier when importing the package. 

We already have subpackages and this is working.

Reply:
My mistake. You're right.